Makefile

```diff
@@ -14,3 +14,11 @@ buildExternalChecker:
 
 pushExternalChecker:
 	docker push integrii/test-external-check:latest
+
+podRestarts: buildPodRestartsCheck pushPodRestartsCheck
+
+buildPodRestartsCheck:
+	docker build -t docker-proto.repo.theplatform.com/pod-restarts-check:1.0.0 -f cmd/podRestartsExternalCheck/Dockerfile .
+
+pushPodRestartsCheck:
+	docker push docker-proto.repo.theplatform.com/pod-restarts-check:1.0.0
```

Comment:
need to build and push somewhere public

Reply:
let's wait for this image to be hosted in our quay.io account before merging so we don't forget